src/source/generator.scala

```diff
@@ -137,6 +138,22 @@ package object generatorTools {
     }
   }
 
+  object JavaAccessModifier extends Enumeration {
+    val Public = Value("public")
+    val Protected = Value("protected")
```

Comment:
I don't think that "protected" is a valid value for top-level classes.  It would be meaningless, so I think it's disallowed.

http://stackoverflow.com/questions/3869556/why-a-class-cannot-be-defined-as-protected

I think public and package-private are the only values this option needs to support.